nautobot/extras/querysets.py

```diff
@@ -70,14 +65,17 @@ class ConfigContextModelQuerySet(RestrictedQuerySet):
 
     def annotate_config_context_data(self):
```

Comment:
Given the changes to `ConfigContextModel.get_config_context()`, `annotate_config_context_data()` is now only used in views. However, we may still need to iterate here to completely remove the dependency ordered JSON aggregation.

Reply:
The reason for this is because in the device & VM views we need the actual config context objects to show other data about each object like its name and weight. Maybe if we move the deepmerge logic around we could get rid of this altogether and use the one method for both use cases?